demos/polls/README.rst

```diff
@@ -7,20 +7,57 @@ similar to django one.
 Installation
 ============
 
-Install the app::
+Prepare development environment
+-------------------------------
+
+We suggest you to start by creating an isolated Python working environment::
+
+    $ python3 -m venv env
+    $ source env/bin/activate
+
+
+Install the app and it's requirements::
 
     $ cd demos/polls
     $ pip install -e .
 
-Create database for your project::
 
-    bash sql/install.sh
+Prepare database
+----------------
+Install Postgresql database server: http://www.postgresql.org/download/
+(to use postgres in more isolated way you may also use docker. We will explain how to do that
+after environment variables setup.)
+
+Create ``.env`` file in ``polls`` folder with db related configuration values. It is a common practice to not
+hardcode such settings because they differ between e.g. prod and dev environments::
+
+    DB_HOST=localhost
+    DB_PORT=5432
+    DB_NAME=aiohttpdemo_polls
+    DB_USER=aiohttpdemo_user
+    DB_PASS=aiohttpdemo_user
```

Comment:
aiohttpdemo_pass

Reply:
it was just intentional copy-paste-as-it-is, but I see your point

Reply:
I mean, copy-paste from existing config file, not between parameters

Reply:
I know (: Leave you this comment just to make you know that this may confuse users about the values kind for these ENVs.